schemas/computation.schema.tpl.json

```diff
@@ -70,6 +71,15 @@
         "type": "string"
       }
     },
+    "technique": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add one or several analysis techniques that were used in this computation.",
```

Comment:
"Add all analysis techniques that were used during this activity."
(in accordance with the clean up)